styx-scheduler-service/src/main/java/com/spotify/styx/TriggerManager.java

```diff
@@ -68,6 +73,7 @@
     this.time = requireNonNull(time);
     this.storage = requireNonNull(storage);
     this.stats = requireNonNull(stats);
+    this.forkJoinPool = new ForkJoinPool(TRIGGER_CONCURRENCY);
```

Comment:
Make the `TriggerManager` `Closeable` and shut down the fjp on scheduler shutdown.

Reply:
Good catch, will fix!

Reply:
fixed by https://github.com/spotify/styx/pull/449/commits/f4a76013669a7e0123bd1c51698bd1956314f3df